docs/data/base/components/modal/modal.md

```diff
@@ -0,0 +1,125 @@
+---
+product: material-ui
+title: React Modal component
+components: Modal, ModalUnstyled
+githubLabel: 'component: modal'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#dialog_modal
+---
+
+# Modal
+
+<p class="description">The `Modal` component lets you create dialogs, popovers, lightboxes, and other windows that force the user to take action before continuing.</p>
+
+The `Modal` component renders its `children` node in front of a backdrop component. This lets you create a window that your users must interact with before continuing in the parent application.
+Features:
+
+- 💄 Manages modal stacking when more than one is needed
+- 🔐 Creates a backdrop to disable interaction with the rest of the app
+- 🔐 Disables page scrolling while open
+- ♿️ Manages focus correctly between the modal and its parent app
+- ♿️ Adds the appropriate ARIA roles automatically
+
+{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+
+> **Note:** the term "modal" is sometimes used interchangeably with "dialog," but this is incorrect.
+> A dialog may be _modal_ or _nonmodal_.
+> 
+> A modal [blocks interaction with the rest of the application](https://en.wikipedia.org/wiki/Modal_window), forcing the user to take action.
+> As such, it should be used sparingly—only when the app _requires_ user input before it can continue.
```

Comment:
This is a good example of why we could use an _info_ type of blockquotes. This information is not a warning per see, but rather a "did you know?" — Anyway, a bit of topic but wanted to highlight that!

Reply:
+1 for that! I believe it was _Docs for Developers_ where I read the suggestion to use three kinds of callouts:

1. Note - interesting point to consider
2. Caution - keep this in mind to avoid problems
3. Warning - this is dangerous, proceed carefully

Reply:
+1 for having these three categories

Reply:
Nice :) To not side-track this, I'll open an issue!